packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -564,7 +560,7 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
 
         <oui-message
             data-type="warning"
-            data-ng-if="$ctrl.subnetGateways.length > 0 && !$ctrl.isGatewayLoading && $ctrl.selectedMode.name !== $ctrl.instanceModeEnum[1].mode"
+            data-ng-if="$ctrl.subnetGateways.length > 0 && !$ctrl.isGatewayLoading && $ctrl.selectedMode.name !== $ctrl.PUBLIC_MODE"
```

Comment:
If we refer to the legacy code, this message should appear if selectedMode is different than PRIVATE_MODE
as  $ctrl.instanceModeEnum[1] was { mode: 'private_mode' }